roxie/topo/toposerver.cpp

```diff
@@ -331,6 +333,9 @@ int main(int argc, const char *argv[])
             queryLogMsgManager()->enterQueueingMode();
             queryLogMsgManager()->setQueueDroppingLimit(512, 32);
         }
+#else
+        updateContainerizedLogMsgHandlerLogFields(MSGFIELD_time | MSGFIELD_milliTime | MSGFIELD_thread | MSGFIELD_prefix);
```

Comment:
we really want these field types in the config and default (in defaultYaml)
e.g. in this component it can have:
```
    logFields: time, milliTime, thread, prefix
```

in it's defaultYaml - processed by loadConfiguration, and then picked up by the standard install log handler routine.